apps/dapp/src/services/MetricsService.ts

```diff
@@ -1,26 +1,23 @@
-//@ts-nocheck
```

Comment:
removing this nocheck revealed a ton of errors in this file, so quite a few general fixes besides migrating away from env vars